ui/perfherder/graphs/TestDataModal.jsx

```diff
@@ -429,6 +429,7 @@ export default class TestDataModal extends React.Component {
 
     return (
       <Modal size="lg" isOpen={showModal}>
+        {this.state.pinnedProjects}
```

Comment:
You can use destructuring assignment for `pinnedProjects`.

Reply:
ah, thanks for catching that. I had already included it in the destructuring assignment above but was still referencing `this.state` directly